src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double firstBase;
+    private double secondBase;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double firstBase, double secondBase, double height) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.height = height;
+    }
+
+    public double getFirstBase() {
+        return firstBase;
+    }
+
+    public double getSecondBase() {
+        return secondBase;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstBase + secondBase) * height) / 2;
```

Comment:
To improve readability, consider assigning the expression for calculating the area to a local variable before returning it. This will make it easier to understand what the method is doing at a glance.